Tools/clinic/cpp.py

```diff
@@ -186,7 +186,7 @@ def _main(filenames: list[str] | None = None) -> None:
             cpp = Monitor(filename, verbose=True)
             print()
             print(filename)
-            for line_number, line in enumerate(f.read().split('\n'), 1):
+            for line in f:
```

Comment:
This line doesn't have any test coverage, but I tested manually, and running `python Tools/clinic/cpp.py` on the command line seems to produce the same output with this change.